serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-redhat-application-services-integration.adoc

```diff
@@ -0,0 +1,93 @@
+* A Service Account.
+
+.Procedure
+. Create a Service Registry instance on Red Hat Openshift Application and Data Services console (if you already have one, skip this step):
+  * Go to link:{openshift_application_data_services_service_registry_url}[Service Registry | Red Hat OpenShift Application and Data Services];
+  * Click on **Create Service Registry instance**;
+  * In the window that opens up type a Service Registry instance name;
+  * Click on **Create**;
+  * The list of instances will be updated with your new instance;
+  * Find it in the list and click on its name;
+  * Go to the **Settings** tab and click on **Grant access**;
+  * From the dropdown, select your Service Account desired (the same you configured on the Serverless Logic Web Tools);
+  * Select a role for that Service Account (has to be Manager or Administrator, to have read and write access);
+  * Click on **Save**;
+  * On the top right-hand corner there should be a triple dotted menu, click on it and then on **Connection**;
+  * A drawer should open with all the connection and authentication information you'll need;
+  * Copy the **Core Registry API** value.
+. If you skipped the previous step, find your Service Registry instance **Core Registry API** as it will be necessary for the next steps;
+. In the Serverless Logic Web Tools, click on the **Cog wheel** on the top right corner and go to the **Service Registry** tab;
+. Input a name for your registry, preferably the same one you used when creating the Service Registry instance;
+. Paste your **Core Registry API** in the respective field;
+. Click on **Apply**.
+. The tab contents should be updated, showing **Your Service Registry information is set.**.
+
+[[proc-setting-apache-kafka-serverless-logic-web-tools]]
+== Setting up Streams for Apache Kafka
+
+Create or use a Kafka instance from your Red Hat OpenShift Application and Data Services console and add it to the Serverless Logic Web Tools settings tab.
+
+.Prerequisites
+* Access to Red Hat OpenShift Application and Data Services console;
+
+.Procedure
+. Create a Kafka instance on Red Hat Openshift Application and Data Services console (if you already have one, skip this step):
+  * Go to link:{openshift_application_data_services_apache_kafka_url}[Streams for Apache Kafka | Red Hat OpenShift Application and Data Services];
+  * Click on **Create Kafka instance**;
+  * In the window that opens up type a Kafka instance name;
+  * Fill the other fields to your liking, or leave them with the default values;
+  * Click on **Create instance**;
+  * Reload the page for the list of instances to be updated with your new instance;
+  * Wait for the status to be updated to **Ready**;
+  * Find it in the list and click on its name;
+  * Go to the **Topics** tab and create a new topic, you'll need its name later;
+  * On the top right-hand corner there should be a triple dotted menu, click on it and then on **Connection**;
+  * Copy the **Bootstrap server** value.
```

Comment:
We would also need additional steps to grant some permissions for the service account.

1. In your Kafka instance, go to "Access" tab
2. Click "Manage Access" and select "All Accounts" or your service account
3. Add the following permissions:
3.1. Consumer group is " * " | Allow All | All Accounts
3.2. Topic is " * " | Allow All | All Accounts

Note: using these broad settings is meant to make it easy to configure but you can set up specific Service Accounts, Topics, and Consumer Groups as well.